service/server.py

```diff
@@ -67,21 +68,38 @@ def get_properties_for_address(address):
     return query.execute().hits
 
 
+def paginated_address_records(address_records, page_number):
+    start_index = (page_number-1)*int(SEARCH_RESULTS_PER_PAGE)
+    end_index = max(page_number*int(SEARCH_RESULTS_PER_PAGE), len(address_records)-1)
+    return format_address_records(address_records[start_index:end_index])
+
+
+def paginated_and_index_address_records(address_records, page_number):
+    if address_records:
+        result = paginated_address_records(address_records, page_number)
+        number_pages = int(len(address_records) / int(SEARCH_RESULTS_PER_PAGE))+1
```

Comment:
The `+1` part should only be present when `len(address_records) % int(SEARCH_RESULTS_PER_PAGE) != 0`